frontend/package.json

```diff
@@ -3,13 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@faker-js/faker": "^7.6.0",
```

Comment:
Love faker libraries! Use them extensively in testing myself (as well as the Insomnia plugin for randomized data in API calls)

Reply:
Yes I use faker all the time in my backend jest e2e/unit tests. Fun library for sure.